tests/GenreSelect.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { render, fireEvent, waitFor } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import GenreSelect from "../src/components/GenreSelect";
+import userEvent from "@testing-library/user-event";
+
+it("renders all genres passed in props", () => {
+  const genres = ["All", "Documentary", "Comedy"];
+  const { getByTestId, getAllByRole } = render(
```

Comment:
it is better to use screen from "@testing-library/react" instead of spreading render result
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen